modules/common/log/loki.nix

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+{
+  config,
+  lib,
+  ...
+}: let
+  loki_data_dir = "/var/lib/loki";
+  cfg = config.ghaf.logging.client;
+  loggingPort = config.ghaf.logging.port;
+in {
+  config = lib.mkIf cfg.enable {
+    systemd.tmpfiles.rules = [
+      "d /var/lib/loki 0777 loki loki - -"
```

Comment:
StateDirectory=loki on the loki systemd service should already do that, no?

Reply:
Not a "bug" necessarily, but having a StateDirectory = loki in that unit file in case cfg.dataDir is /var/lib/loki sounds like a good thing to upstream. We can unconditionally extend the Loki systemd unit file config downstream in the meantime.

Reply:
The nice part is that systemd automatically ensures everything is owned by the user when the service is started.

Reply:
We don't need `loki.service` anymore on device.